RecoTracker/CkfPattern/src/CkfTrackCandidateMakerBase.cc

```diff
@@ -76,34 +76,32 @@ namespace cms {
             iC.esConsumes(edm::ESInputTag("", conf.getParameter<std::string>("NavigationSchool")))),
         theNavigationSchool(nullptr),
         thePropagatorToken(iC.esConsumes(edm::ESInputTag("", "AnyDirectionAnalyticalPropagator"))),
+#ifdef VI_REPRODUCIBLE
         maxSeedsBeforeCleaning_(0),
+#else
+        maxSeedsBeforeCleaning_(conf.getParameter<unsigned int>("maxSeedsBeforeCleaning")),
+#endif
         theMTELabel(iC.consumes<MeasurementTrackerEvent>(conf.getParameter<edm::InputTag>("MeasurementTrackerEvent"))),
-        skipClusters_(false),
-        phase2skipClusters_(false) {
+        clustersToSkipTag_(conf.getParameter<edm::InputTag>("clustersToSkip")),
+        skipClusters_(!clustersToSkipTag_.label().empty()),
+        phase2ClustersToSkipTag_(conf.getParameter<edm::InputTag>("phase2clustersToSkip")),
```

Comment:
Since `clustersToSkip` and `phase2ClustersToSkip` are supposed to be used alternatively, I have the impression that these settings could be simplified... But I agree this is not the PR where to do it, you had already quite a lot of adjustments to apply here only to implement fillDescriptions (thank you for it, by the way!)